corehq/ex-submodules/casexml/apps/phone/restore_caching.py

```diff
@@ -0,0 +1,75 @@
+import hashlib
+import logging
+from casexml.apps.phone.const import RESTORE_CACHE_KEY_PREFIX, ASYNC_RESTORE_CACHE_KEY_PREFIX
+from dimagi.utils.couch.cache.cache_core import get_redis_default_cache
+
+logger = logging.getLogger(__name__)
+
+
+def _restore_cache_key(domain, prefix, user_id, version, sync_log_id, device_id):
+    # to invalidate all restore cache keys, increment the number below
+    hashable_key = '8-{domain}-{prefix}-{user}-{version}-{sync_log_id}-{device_id}'.format(
+        domain=domain,
+        prefix=prefix,
+        user=user_id,
+        version=version or '',
+        sync_log_id=sync_log_id or '',
+        device_id=device_id or '',
+    )
+    return hashlib.md5(hashable_key).hexdigest()
+
+
+def _restore_payload_path_cache_key(domain, user_id, sync_log_id, device_id):
+    return _restore_cache_key(
+        domain=domain,
+        prefix=RESTORE_CACHE_KEY_PREFIX,
+        user_id=user_id,
+        version='2.0',
```

Comment:
what does this version mean? why is it hard-coded here rather than using the version specified in the restore config?